src/util/markdown.ts

```diff
@@ -28,5 +28,5 @@ marked.use({
 
 export function markdown(input: string) {
 	const content = marked(input);
-	return content.replace(/<(info|warn)>([\s\S].+)<\/\1>/gi, '<div class="$1">$2</div>');
+	return content.replace(/<(info|warn)>(.+\n*.+)<\/\1>/gi, '<div class="$1">$2</div>');
```

Comment:
This works with multiple line breaks unlike the suggested regex which only works with one
```suggestion
	return content.replace(/<(info|warn)>([\s\S]+)<\/\1>/gi, '<div class="$1">$2</div>');
```

Reply:
Reverts back to the original which didn't work with multiple blocks, but with a lazy quantified it does, so that's what I've done